app/models/entry.rb

```diff
@@ -28,7 +28,7 @@ def process_entry
   end
 
   def self.for_model(model)
-    new(model: model, tenant: model.try!(:tenant))
+    new(model: model, tenant: model&.tenant)
```

Comment:
@mayorova , this is not the same. `try!` should ignore any call errors while `&.tenant` will raise if `model` is not `nil` and `#tenant` method does not exist.

Reply:
@akostadinov I've checked this, and what you're saying is true for `.try`, on the other hand `.try!` does throw a `NoMethodError`, so its behavior is identical to `&.`